.github/workflows/pr_test_build_android.yml

```diff
@@ -93,6 +93,33 @@ jobs:
           cd /opt/android/cake_wallet
           flutter pub get
 
+
+      - name: Install go and gomobile
+        run: |
+          # install go > 1.23:
+          wget https://go.dev/dl/go1.23.1.linux-amd64.tar.gz
+          sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.23.1.linux-amd64.tar.gz
+          export PATH=$PATH:/usr/local/go/bin
+          export PATH=$PATH:~/go/bin
+          go install golang.org/x/mobile/cmd/gomobile@latest
+          gomobile init
+
+      - name: Build mwebd
+        run: |
+          # paths are reset after each step, so we need to set them again:
```

Comment:
I had the same thought but the script is ~slightly~ different
I can just add an option to the script though

Reply:
just to add: mweb on linux will need some work since gomobile doesn't support linux as a target unfortunately (it does look like it supports macos though)
so we'll need to figure out a different way to generate bindings for those platforms